### PR TITLE
Implement builtin getters for Date object

### PR DESCRIPTION
**Pull request by szledan**
JerryScript-DCO-1.0-Signed-off-by: Szilard Ledan szledan.u-szeged@partner.samsung.com

### CODE REVIEW

File: jerry-core/ecma/builtin-objects/ecma-builtin-date-prototype.cpp [hunk 1]
```diff
@@ -154,248 +158,92 @@ ecma_builtin_date_prototype_value_of (ecma_value_t this_arg) /**< this argument
...
+ */
+#define DEFINE_GETTER(_ecma_paragraph, _routine_name, _getter_name, _timezone) \
+static ecma_completion_value_t \
+ecma_builtin_date_prototype_get_ ## _routine_name (ecma_value_t this_arg) /**< this argument */ \
+{ \
+  ecma_completion_value_t ret_value = ecma_make_empty_completion_value (); \
+ \
+  /* 1. */ \
+  ECMA_TRY_CATCH (value, ecma_builtin_date_prototype_get_time (this_arg), ret_value); \
+  ecma_number_t *this_num = ecma_get_number_from_value (value); \
```
Comment (egavrin) [inline jerry-core/ecma/builtin-objects/ecma-builtin-date-prototype.cpp]: `_p`

File: jerry-core/ecma/builtin-objects/ecma-builtin-date-prototype.cpp [hunk 2]
```diff
@@ -409,8 +257,30 @@ ecma_builtin_date_prototype_get_utc_milliseconds (ecma_value_t this_arg) /**< th
 static ecma_completion_value_t
 ecma_builtin_date_prototype_get_timezone_offset (ecma_value_t this_arg) /**< this argument */
 {
-  ECMA_BUILTIN_CP_UNIMPLEMENTED (this_arg);
-} /* ecma_builtin_date_prototype_get_timezone_offset */
+  ecma_completion_value_t ret_value = ecma_make_empty_completion_value ();
```
Comment (egavrin) [inline jerry-core/ecma/builtin-objects/ecma-builtin-date-prototype.cpp]: Looks like, the macro body above. Am I missing smth?

Comment (zherczeg) [inline jerry-core/ecma/builtin-objects/ecma-builtin-date-prototype.cpp]: I think we can move the time_offset computation into a function. That would clean up things.

Comment (egavrin) [inline jerry-core/ecma/builtin-objects/ecma-builtin-date-prototype.cpp]: +1

**Comment by szledan** [comment]
Hi, I've updated this PR.

**Comment by egavrin** [comment]
Good to me.

**Comment by kkristof** [comment]
Could you add some test? Otherwise the patch looks good.

**Comment by szledan** [comment]
Sure

**Comment by kkristof** [comment]
lgtm

**Comment by egavrin** [comment]
`make push`

**Pull request merged**